generators/app/templates/src/app/index.ejs

```diff
@@ -22,10 +21,8 @@ const App = () => {
     <%_ if(features.pushnotifications) { _%>
     configPushNotifications();
     <%_ } _%>
-    <%_ if(features.login) { _%>
     authSetup(dispatch);
-    <%_ } _%>
-    apiSetup();
+    apiSetup(dispatch);
```

Comment:
To always send the dispatch, because The most of the times you will want to dispatch actions from the monitors. I think it's good to be prepare for that